ansible-scylla-node/tasks/repair.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Start repair
+  uri:
+    url: "http://{{ scylla_api_address }}:{{ scylla_api_port }}/storage_service/repair_async/{{ keyspace }}?primaryRange='true'"
+    method: POST
+  register: _repair_id
+  until: _repair_id.status == 200
+  retries: 5
+  delay: 1
+
+- name: Wait for the repair to finish
+  uri:
+    url: "http://{{ scylla_api_address }}:{{ scylla_api_port }}/storage_service/repair_async/{{ keyspace }}?id={{ _repair_id.json }}"
+    method: GET
+  register: _repair_status
+  until: _repair_status.json != 'RUNNING'
+  retries: 1200 # 1200 * 30 seconds = 10 hours
```

Comment:
Let's parametrize it: create a variable that would hold a corresponding delay and create a reasonable default (e.g. 10h as here).